DiagManager/CustomDiagnostics/SQL Base/MiscPssdiagInfo.sql

```diff
@@ -254,8 +146,14 @@ order by PropertyName
 drop table #summary
 go
 
+--changing xp_instance_regenumvalues to dmv access as a part of issue #149
+
 declare @startup table (ArgsName nvarchar(10), ArgsValue nvarchar(max))
-insert into @startup EXEC master..xp_instance_regenumvalues 'HKEY_LOCAL_MACHINE',   'SOFTWARE\Microsoft\MSSQLServer\MSSQLServer\Parameters'
+insert into @startup 
+SELECT     sReg.value_name,     CAST(sReg.value_data AS nvarchar(max))
+FROM sys.dm_server_registry AS sReg
+WHERE     sReg.registry_key LIKE N'%Parameters';
```

Comment:
Though this is a very small table, and scans don't matter, just to keep practicing good coding practices and avoid index scanning, I am proposing the following rewrite of the WHERE clause. You can just press the Commit button under the comment to commit the change in your project and then pull them down locally (git pull) 

```suggestion
WHERE     sReg.value_name LIKE N'SQLArg%';
```